homeassistant/components/ecobee/climate.py

```diff
@@ -741,6 +774,59 @@ def resume_program(self, resume_all):
         )
         self.update_without_throttle = True
 
+    def set_sensors_used_in_climate(
+        self, sensors: list[str], preset_mode: str | None = None
+    ) -> None:
+        """Set the sensors used on a climate for a thermostat."""
+        if preset_mode is None:
+            preset_mode = self.preset_mode
```

Comment:
Add a test for this case (updating sensors without changing the preset mode)

Reply:
I will add this test with the next commits